quill-jdbc-zio/src/main/scala/io/getquill/jdbczio/Quill.scala

```diff
@@ -98,16 +98,16 @@ object Quill {
   }
 
   object Connection {
-    def acquireScoped: ZLayer[DataSource, SQLException, Connection] =
+    val acquire: ZLayer[DataSource, SQLException, Connection] =
```

Comment:
Renamed `acquire` because the returned type doesn't need a `Scope`
The Scope needs is only an internal requirement that the external world doesn't need to know about